koku/reporting/migrations/0213_delete_aws_mat_views.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.13 on 2021-12-08 18:38
+from django.db import migrations
+
+
```

Comment:
```suggestion
from koku.database import get_model


SQL_TMPL = """DROP MATERIALIZED VIEW IF EXISTS {} CASCADE ;"""


def drop_matview_sql(model_str):
    return SQL_TMPL.format(get_model(model_str)._meta.db_table)


```

Reply:
The `get_model` was not working for matviews, so I pass in the model name instead.